content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/cpu-manager-policy-alpha-options.md

```diff
@@ -0,0 +1,16 @@
+---
+title: CPUManagerPolicyAlphaOptions
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+---
+<!--
+This allows fine-tuning of CPUManager policies,
+experimental, Alpha-quality options
+This feature gate guards *a group* of CPUManager options whose quality level is alpha.
+This feature gate will never graduate to beta or stable.
+-->
+允许对 CPUManager 策略进行微调，针对试验性的、Alpha 质量级别的选项。
+此特性门控用来保护一组质量级别为 Alpha 的 CPUManager 选项。
```

Comment:
I'm not sure what the current page says, but IMO it's OK to localize “CPU manager”.